gateway/src/connectors/uniswap/uniswap.controllers.ts

```diff
@@ -187,7 +188,7 @@ export async function price(
     gasPrice: gasPrice,
     gasPriceToken: ethereumish.nativeTokenSymbol,
     gasLimit: gasLimit,
```

Comment:
Considering that now the `gasCost` is calculated with `gasEstimate` instead of `gasLimit`, would it make sense to include `gasEstimate` as part of the return value?

Reply:
The client doesn't need it and it's not necessary. The cost also outweighs the benefit. We'd need to modify interfaces and update files and tests.